integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/DomainUtils.java

```diff
@@ -139,4 +225,507 @@ public static void deleteDomainResource(String domainNS, String domainUid) {
+        clusterName, replicaCount);
+  }
+
+  /**
+   * Create domain with domain-in-image type and verify the domain is created.
+   * Also verify the admin server pod and managed server pods are up and running.
+   * @param domainUid domain uid
+   * @param domainNamespace domain namespace
+   * @param imageName image name used to create domain-in-image domain
+   * @param wlSecretName wls admin secret name
+   * @param clusterName cluster name
+   * @param replicaCount replica count of the cluster
+   * @return oracle.weblogic.domain.Domain object
+   */
+  public static Domain createDomainInImageAndVerify(String domainUid,
+                                                    String domainNamespace,
+                                                    String imageName,
+                                                    String wlSecretName,
+                                                    String clusterName,
+                                                    int replicaCount) {
+    // create the domain custom resource
+    Domain domain = new Domain()
+        .apiVersion(DOMAIN_API_VERSION)
+        .kind("Domain")
+        .metadata(new V1ObjectMeta()
+            .name(domainUid)
+            .namespace(domainNamespace))
+        .spec(new DomainSpec()
+            .domainUid(domainUid)
+            .domainHome(WDT_IMAGE_DOMAINHOME_BASE_DIR + "/" + domainUid)
+            .dataHome("/u01/oracle/mydata")
+            .domainHomeSourceType("Image")
+            .image(imageName)
+            .addImagePullSecretsItem(new V1LocalObjectReference()
+                .name(OCIR_SECRET_NAME))
+            .webLogicCredentialsSecret(new V1SecretReference()
+                .name(wlSecretName)
+                .namespace(domainNamespace))
+            .includeServerOutInPodLog(true)
+            .serverStartPolicy("IF_NEEDED")
+            .serverPod(new ServerPod()
+                .addEnvItem(new V1EnvVar()
+                    .name("JAVA_OPTIONS")
+                    .value("-Dweblogic.StdoutDebugEnabled=false"))
+                .addEnvItem(new V1EnvVar()
+                    .name("USER_MEM_ARGS")
+                    .value("-Djava.security.egd=file:/dev/./urandom ")))
+            .adminServer(new AdminServer()
+                .serverStartState("RUNNING")
+                .adminService(new AdminService()
+                    .addChannelsItem(new Channel()
+                        .channelName("default")
+                        .nodePort(getNextFreePort()))))
+            .addClustersItem(new Cluster()
+                .clusterName(clusterName)
+                .replicas(replicaCount)
+                .serverStartState("RUNNING"))
+            .configuration(new Configuration()
+                .model(new Model()
+                    .domainType(WLS_DOMAIN_TYPE))
+                .introspectorJobActiveDeadlineSeconds(300L)));
+    setPodAntiAffinity(domain);
+    createDomainAndVerify(domain, domainNamespace);
```

Comment:
call  createDomainAndVerify(String domainUid, Domain domain,
                                           String domainNamespace, String adminServerPodName,
                                           String managedServerPodNamePrefix, int replicaCount) and remove the code to check pods

Reply:
fixed